### PR TITLE
 add support for SPLUNK_TRACE_RESPONSE_HEADER_ENABLED env variable 

### PR DESCRIPTION
**Pull request by tsloughter-splunk**

### CODE REVIEW

**Review by aurbiztondo-splunk — APPROVED** [review]

File: lib/splunk/otel.rb [hunk 1]
```diff
@@ -151,6 +158,20 @@ def default_env_vars(env)
       end
     end
 
+    def to_boolean(val)
```
Comment (jtmalinowski) [inline lib/splunk/otel.rb:161]: It's a pity that OTel doesn't export this so we could reuse.

File: README.md [hunk 2]
```diff
@@ -187,6 +187,46 @@ Splunk::Otel.configure do |c|
...
+    run ->(_env) { [200, { "content-type" => "text/plain" }, ["OK"]] }
+end
+```
+
+When using [ActionPack](https://rubygems.org/gems/actionpack/), which Rails
+does, the middleware will be added automatically if the Instrumentation Library
+for ActionPack, "Splunk::Otel::Instrumentation::ActionPack", is used:
+
+``` ruby
+Splunk::Otel.configure do |c|
```
Comment (jtmalinowski) [inline README.md:220]: @aurbiztondo-splunk should we mention that `auto_instrument: true` also works? (and I think it'll be a slightly more popular option)

**Review by jtmal-signalfx — APPROVED** [review]
added some nitpicky comments but otherwise looks fine

**Review by jtmal-signalfx — APPROVED** [review]
added some nitpicky comments but otherwise looks fine

**Comment by tsloughter-splunk** [comment]
I'll go ahead and merge and open the gdi-spec issue for approval, we can update the docs if need be in another PR.

**Pull request merged**